src/newsletter-editor/editor/index.js

```diff
@@ -133,6 +142,23 @@ const Editor = compose( [
 		}
 	}, [ props.status ] );
 
+	// Notify if email content is larger than ~100kb.
+	useEffect( () => {
+		const noticeId = 'newspack-newsletters-email-content-too-large';
+		const message = __(
+			'Email content is too large and may get clipped by email clients.',
```

Comment:
Nit: "long" sounds to me like it better suggests what action the editor should take to fix this error.

```suggestion
			'Email content is too long and may get clipped by email clients.',
```